server/server.go

```diff
@@ -2890,6 +2922,11 @@ func (l decisionLogger) Log(ctx context.Context, txn storage.Transaction, decisi
 		Metrics:    m,
 	}
 
+	if ndbCache != nil {
+		var x interface{} = ast.MustJSON(ndbCache.AsValue())
+		info.NDBuiltinCache = &x
+	}
+
```

Comment:
This snippet is responsible for transforming an `NDBCache` into a `map[string]interface{}` that the masking system will know how to work on later in the logging pipeline.

Reply:
I don't know, we don't need to overdo it. The output of http.send will be an object, so I guess we're just OK here -- anything wonky in its raw_body key for example will be escaped accordingly. Your call 😄